src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -14,10 +14,12 @@ import StatusAlert from '../StatusAlert';
 import './styles/SubscriptionManagementPage.scss';
 import { TAB_PENDING_USERS } from './constants';
 import AddUsersButton from './AddUsersButton';
-import { configuration } from '../../config';
 
 const PAGE_TITLE = 'Subscription Management';
-const { CUSTOMER_SUPPORT_EMAIL } = configuration;
+
+// get current url and remove the 'subscription'
+const CURRENT_URL = window.location.pathname;
+const URL_BASE = CURRENT_URL.split('/').slice(0, -1).join('/');
```

Comment:
Instead of parsing the URL to get the enterprise's slug, I'm curious if we could use react-router's [`useParams` hook](https://reacttraining.com/react-router/web/api/Hooks/useparams) to get the `enterpriseSlug` from the route props?